scripts/test/shared.py

```diff
@@ -54,6 +54,10 @@ def parse_args(args):
               ' Default: bin/ of current directory (i.e. assume an in-tree build).'
               ' If not specified, the environment variable BINARYEN_ROOT= can also'
               ' be used to adjust this.'))
+    parser.add_argument(
+        '--binaryen-build', dest='binaryen_build', default='',
+        help=('Specifies a path to the Binaryen build directory. '
+              'Default: parent of `binaryen_bin`'))
```

Comment:
Why do we need this in addition `--binaryen-bin`?   It would be nice if we could avoid adding more arguments here.

Reply:
Answered [here](https://github.com/WebAssembly/binaryen/pull/3367#issuecomment-727641092) so we can keep this discussion focused more on the wasm-split code.